### PR TITLE
Update first_steps.md

### PR DESCRIPTION
**Pull request by Alwoch**
The current documentation seems to imply that we are updating snap. I just had to specify that its the lxd snap we're updating to avoid confusion

### CODE REVIEW

File: doc/tutorial/first_steps.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ If you prefer a different installation method, or use a Linux distribution that
 
        sudo snap install lxd
 
-   If you get an error message that the snap is already installed, run the following command to refresh it and ensure that you are running an up-to-date version:
+   If you get an error message that the lxd snap is already installed, run the following command to refresh it and ensure that you are running an up-to-date version:
```
Comment (MggMuggins) [inline doc/tutorial/first_steps.md:49]: Looks like capital "LXD" is the convention here

**Review by MggMuggins — CHANGES_REQUESTED** [review]
One nit :)

**Comment by MggMuggins** [comment]
Also can you please sign [Canonical's CLA](https://ubuntu.com/legal/contributors)?

**Comment by Alwoch** [comment]
> Also can you please sign [Canonical's CLA](https://ubuntu.com/legal/contributors)?

Done! also I've it to conform to the convention

**Comment by MggMuggins** [comment]
Thanks; can you also rebase the PR into one commit and use the commit title convention that's described [here](https://documentation.ubuntu.com/lxd/en/latest/contributing/#commit-structure)?

**Comment by simondeziel** [comment]
`git rebase -i main` should be all you need to then mark the last commit (`capitalize LXD`) for `fixup`. This will "merge" or "fuse" the 2 commits into one.

After that, you should be able to push with force: `git push -f`

**Comment by tomponline** [comment]
Hi @minaelee please could you assist with either getting this contribution ready to be merged, or potentially make the change yourself and we can close this? Thanks

**Comment by minaelee** [comment]
@Alwoch Thanks for your contribution! I see that it's your first contribution to Canonical, and I'd love to help you get it ready for merge.

Here are detailed instructions to rebase as well as to update the commit message:
- In a terminal, in a local clone of your fork of the lxd repository, switch to your PR branch (`patch-1`). (Note: if you used the GH web interface to make your commits and PR instead of a local clone, you'll need to clone your forked repository locally, checkout/switch to the PR branch there (`git switch -c patch-1 origin/patch-1`), then sync with `git pull origin patch-1` before you rebase.)
- Then run `git rebase -i main` in your terminal. This will open the text editor set for git, or the default system editor if one isn't set for git.
- In the text editor, you should see a list of your recent commits. To the left of the last commit, change the text `pick` to `fixup`. Save & exit the editor.
- Run `git commit --amend`. This will open the editor again. In it, edit your commit message to something like `doc/tutorial: specify LXD snap` and save & exit.
- Force push the changes with `git push -f`. 

Reach out if you have questions. In the interest of completing this change, I will wait a week and if I haven't heard back, I will close this PR and open a new one with attribution to you.

**Comment by Alwoch** [comment]
> @Alwoch Thanks for your contribution! I see that it's your first contribution to Canonical, and I'd love to help you get it ready for merge.
> 
> Here are detailed instructions to rebase as well as to update the commit message:
> 
> * In a terminal, in a local clone of your fork of the lxd repository, switch to your PR branch (`patch-1`). (Note: if you used the GH web interface to make your commits and PR instead of a local clone, you'll need to clone your forked repository locally, checkout/switch to the PR branch there (`git switch -c patch-1 origin/patch-1`), then sync with `git pull origin patch-1` before you rebase.)
> * Then run `git rebase -i main` in your terminal. This will open the text editor set for git, or the default system editor if one isn't set for git.
> * In the text editor, you should see a list of your recent commits. To the left of the last commit, change the text `pick` to `fixup`. Save & exit the editor.
> * Run `git commit --amend`. This will open the editor again. In it, edit your commit message to something like `doc/tutorial: specify LXD snap` and save & exit.
> * Force push the changes with `git push -f`.
> 
> Reach out if you have questions. In the interest of completing this change, I will wait a week and if I haven't heard back, I will close this PR and open a new one with attribution to you.

Hi @minaelee , you can go ahead and open a new PR because I'm avoiding cloning the repository.  Thanks

**Comment by minaelee** [comment]
Closing in favor of #14382; thanks @Alwoch!

**Pull request closed without merging**